docs/document/docs/guides/table-readonly-table-customization.md

```diff
@@ -0,0 +1,514 @@
+---
+id: table-readonly-table-customization
+title: Table and readOnlyTable customization
+sidebar_label: Table and readOnlyTable customization
+---
+
+## Overview
+This article explains how to utilize and customize the Table, ReadOnlyTable component and the usage difference between them.<br>
+We assume the following scenario:
+1. Table and ReadOnlyTable usage difference
+2. Table and ReadOnlyTable: Create simple one
+3. Table: Set multiple components in one cell
+4. Table: Update cell value depending on the other cell value change
+5. Table: Table in table
```

Comment:
How about unifying the number and title in the detailed section?

Reply:
How about new order ?
<img width="1194" alt="Screen Shot 2023-02-01 at 10 18 13" src="https://user-images.githubusercontent.com/48341495/215938477-c5ecf861-411c-4893-8791-05942a6a1003.png">

Reply:
My comment was not good. I will explain it again.
I think the explanation order is ok.
I mean that "Overview > scenario" title and "Examples" title are different, so it will be easier to understand for users if it is unified.
How do you think?

Reply:
<img width="1020" alt="Screen Shot 2023-02-01 at 13 46 27" src="https://user-images.githubusercontent.com/48341495/215971384-ca4710f4-2c0c-4315-a9b2-f2a6dd6190d5.png">

is it ok with the above update for the scenario and examples?

Reply:
Thanks!
Please choose from these two options.
- Only write Example code scenarios on scenario list (if this, need deleting "usage difference")
- Write full agenda on scenario list (if this, need adding "Understanding change event")

Reply:
Thank for these options.
I will update following option 1 (deleting "usage difference")

Reply:
Got it!